ods_ci/tasks/Resources/Provisioning/Hive/provision.robot

```diff
@@ -210,9 +210,8 @@ Wait For Cluster To Be Ready
         ${provision_status} =    Run Process    oc -n ${pool_namespace} get cd ${clusterdeployment_name} -o json    shell=yes    # robocop: disable:line-too-long
         ${custer_status} =    Run Process    oc -n ${hive_namespace} get clusterclaim ${claim_name} -o json    shell=yes
         Log    Cluster '${cluster_name}' deployment had errors, see: ${\n}${provision_status.stdout}${\n}${custer_status.stdout}    level=ERROR    # robocop: disable:line-too-long
-        Log    Cluster '${cluster_name}' install completed, but it is not accessible - Cleaning Hive resources now
+        Log    Cluster '${cluster_name}' install completed, but it is not accessible
         ...    console=True
-        Deprovision Cluster
```

Comment:
Maybe you can remove the Keywork `Deprovision Cluster` and `Deprovision Self-Managed Cluster` because no one uses them

Reply:
talk offline

Reply:
It looks better 
Thank you!!